### PR TITLE
Add list of supported timezone in help command

### PR DESCRIPTION
**Pull request by jeevanshah07**

### CODE REVIEW

**Comment by TheOtherUnknown** [comment]
Please use the wiki instead of a gist so the information can be updated later.

File: src/cogs/utils.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ async def verify(self, ctx):
             )
 
     @commands.command(
-        usage="[time] [am/pm] [original timezone] [timezone to convert to]")
+        usage="[time] [am/pm] [original timezone] [timezone to convert to]", help="For a list of timezones see: https://github.com/TheOtherUnknown/Malcolm-next/wiki/Commands#time")
     async def time(self, ctx, utime, apm, ozone, nzone):
         """Converts times from one timezone to another"""
```
Comment (TheOtherUnknown) [inline src/cogs/utils.py]: This doc comment is being ignored by the help command. Use it instead of `help=`.

**Review by TheOtherUnknown — CHANGES_REQUESTED** [review]
Help text needs adjustment

File: src/cogs/utils.py (hunk 1 above)
Comment (jeevanshah07) [reply src/cogs/utils.py]: Fixed

**Review by TheOtherUnknown — APPROVED** [review]

**Pull request merged**